Travis build failed: log too long

Note: cmake fails with `-DCMAKE_CXX_FLAGS="-Wno-unused-local-typedefs -Wno-deprecated-declarations`:

```
-- Configuring done                                 
CMake Error in src/CMakeLists.txt:                  
  No known features for CXX compiler                

  "GNU"                                             

  version 7.3.1. 
```

but it accepts `-DCMAKE_C_FLAGS="-Wno-unused-local-typedefs -Wno-deprecated-declarations`, which is useless because the warnings are in C++ code.

I'm investigating.

--- a/Magics.spec
+++ b/Magics.spec
@@ -99,7 +99,7 @@ mkdir build
 pushd build
 
 %cmake .. \
-    -DCMAKE_CXX_FLAGS"$CXXFLAGS -Wno-deprecated-declarations -Wno-deprecated-declarations" \
+    -DCMAKE_CXX_FLAGS="$CXXFLAGS -Wno-deprecated-declarations -Wno-deprecated-declarations" \
     -DCMAKE_PREFIX_PATH=%{_prefix} \
     -DCMAKE_INSTALL_PREFIX=%{_prefix} \
     -DINSTALL_LIB_DIR=%{_lib} \

--- a/Magics.spec
+++ b/Magics.spec
@@ -99,7 +99,7 @@ mkdir build
 pushd build
 
 %cmake .. \
-    -DCMAKE_CXX_FLAGS="$CXXFLAGS -Wno-deprecated-declarations -Wno-deprecated-declarations" \
+    -DCMAKE_CXX_FLAGS="$CXXFLAGS -Wno-deprecated-declarations -Wno-unused-local-typedefs" \
     -DCMAKE_PREFIX_PATH=%{_prefix} \
     -DCMAKE_INSTALL_PREFIX=%{_prefix} \
     -DINSTALL_LIB_DIR=%{_lib} \

--- a/Magics.spec
+++ b/Magics.spec
@@ -99,6 +99,7 @@ mkdir build
 pushd build
 
 %cmake .. \
+    -DCMAKE_CXX_FLAGS"$CXXFLAGS -Wno-deprecated-declarations -Wno-deprecated-declarations" \
     -DCMAKE_PREFIX_PATH=%{_prefix} \
     -DCMAKE_INSTALL_PREFIX=%{_prefix} \
     -DINSTALL_LIB_DIR=%{_lib} \

